pennylane_lightning/core/src/simulators/lightning_kokkos/catalyst/LightningKokkosSimulator.hpp

```diff
@@ -63,6 +63,8 @@ class LightningKokkosSimulator final : public Catalyst::Runtime::QuantumDevice {
 
     std::size_t device_shots;
 
+    std::mt19937 *gen = nullptr;
```

Comment:
Let's use uniform initialization over C-style init with assignment: 
```suggestion
    std::mt19937 *gen{nullptr};
```

Reply:
done